src/analytics/Properties.tsx

```diff
@@ -1499,20 +1497,24 @@ interface TransactionDetailsProperties {
   }
 }
 
-interface JumpstartSendProperties {
+interface JumpstartDepositProperties {
+  amountInUsd: string | null
+  tokenId: string | null
+  networkId: string | null
+}
+interface JumpstartSendProperties extends JumpstartDepositProperties {
   localCurrency: LocalCurrencyCode
   localCurrencyExchangeRate: string | null
   tokenSymbol: string
   tokenAmount: string | null
-  amountInUsd: string | null
-  tokenId: string | null
-  networkId: string | null
+}
+export enum JumpstartShareOrigin {
+  QR = 'qrScreen',
+  MAINSCREEN = 'mainScreen',
```

Comment:
nit: follows the [TS examples](https://www.typescriptlang.org/docs/handbook/enums.html) for naming enums:
```suggestion
  QrScreen = 'qrScreen',
  MainScreen = 'mainScreen',
```